mount-helper/install/install.sh

```diff
@@ -213,14 +213,17 @@ _install_app() {
 _install_apps() { 
     apps=($@)
 
-    if is_linux LINUX_UBUNTU; then
+    if is_linux LINUX_UBUNTU && ([[ ! "$INSTALL_ARG" = "--update" ]]); then
```

Comment:
This syntax needs to be fixed

Reply:
Even in case install_arg is `--update`, the pre_installed_list is getting updated leeding to issues in uninstall